subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/JvmComponentDependencies.java

```diff
@@ -125,4 +125,15 @@ public interface JvmComponentDependencies {
      * @since 7.6
      */
     Dependency testFixtures(ModuleDependency moduleDependency);
+
+    /**
+     * Create a dependency on the current project's complete view. During compile-time, this dependency will
+     * resolve the current project's implementation in addition to its API. During runtime, this dependency
+     * behaves as a usual project dependency.
+     *
+     * @return A new dependency
+     *
+     * @since 7.6
+     */
+    ModuleDependency projectComplete();
```

Comment:
This is a `ProjectDependency`, right? I would prefer to be as specific as possible in the type.